cvat-core/src/frames.js

```diff
@@ -341,13 +341,29 @@
         constructor(size, chunkSize, stopFrame, taskID) {
             this._size = size;
             this._buffer = {};
+            this._contextImage = {};
```

Comment:
Is the contextImage a 3D images, right? Why we need separated data structures for them?

Reply:
We are storing the context image per frame basis in this variable as an entity format for quick retrieval.